spec/requests/session_controller_spec.rb

```diff
@@ -1832,7 +1832,7 @@ def login_with_sso_and_invite(invite_key = invite.invite_key)
       end
 
       it "prevents login by admins" do
-        post "/session.json", params: { login: user.username, password: "myawesomepassword" }
+        post "/session.json", params: { login: admin.username, password: "myawesomepassword" }
         expect(response.status).not_to eq(200)
```

Comment:
Feels like we should test an exact status code here, instead of NOT 200. could be 204 for example and would still be considered "success".